Integrations/SlashNextPhishingIncidentResponse/SlashNextPhishingIncidentResponse.py

```diff
@@ -0,0 +1,975 @@
+    """
+    snx_ioc_cont = {
+        'Value': indicator,
+        'Type': ioc_type,
+        'Verdict': threat_data.get('verdict'),
+        'ThreatStatus': threat_data.get('threatStatus'),
+        'ThreatType': threat_data.get('threatType'),
+        'ThreatName': threat_data.get('threatName'),
+        'FirstSeen': threat_data.get('firstSeen'),
+        'LastSeen': threat_data.get('lastSeen')
+    }
+
+    return snx_ioc_cont
+
+
+def get_snx_url_ioc_context(url_data, is_scan=False):
+    """
+    Make the dictionary for SlashNext URL IoC contexts for URLs
+    :param url_data: URL data received in json format
+    :param is_scan: Is Scan ID to be included
+    :return: List of SlashNext IoC context dictionaries, Entry context dictionary
+    """
+    snx_ioc_cont_list = []
+    dbot_score_cont_list = []
+    url_cont_list = []
+
+    snx_ioc_cont = {
+        'Value': url_data.get('url'),
+        'Type': 'Scanned URL',
+        'Verdict': url_data.get('threatData').get('verdict'),
+        'ThreatStatus': url_data.get('threatData').get('threatStatus'),
+        'ThreatType': url_data.get('threatData').get('threatType'),
+        'ThreatName': url_data.get('threatData').get('threatName'),
+        'FirstSeen': url_data.get('threatData').get('firstSeen'),
+        'LastSeen': url_data.get('threatData').get('lastSeen')
+    }
+    if is_scan is True:
+        snx_ioc_cont['ScanID'] = url_data.get('scanId')
+
+    dbot_score_cont, url_cont = get_dbot_std_context(
+        url_data.get('url'), 'url',
+        url_data.get('threatData').get('verdict'),
+        url_data.get('threatData').get('threatType'))
+    dbot_score_cont_list.append(dbot_score_cont)
+    if url_cont is not None:
+        url_cont_list.append(url_cont)
+
+    if url_data.get('landingUrl') is None:
+        if url_data.get('finalUrl') is not None and url_data.get('finalUrl') != 'N/A':
+            dbot_final_score_cont, final_url_cont = get_dbot_std_context(
+                url_data.get('finalUrl'), 'url',
+                url_data.get('threatData').get('verdict'),
+                url_data.get('threatData').get('threatType'))
+            dbot_score_cont_list.append(dbot_final_score_cont)
+            if final_url_cont is not None:
+                url_cont_list.append(final_url_cont)
+
+            snx_final_ioc_cont = {
+                'Value': url_data.get('finalUrl'),
+                'Type': 'Final URL',
+                'Verdict': url_data.get('threatData').get('verdict')
+            }
+
+            snx_ioc_cont['Final'] = snx_final_ioc_cont.copy()
+            snx_ioc_cont_list.append(snx_ioc_cont)
+
+            snx_final_ioc_cont['Value'] = '--------> {}'.format(url_data.get('finalUrl'))
+            snx_ioc_cont_list.append(snx_final_ioc_cont)
+
+        else:
+            snx_ioc_cont_list.append(snx_ioc_cont)
+
+    else:
+        dbot_landing_score_cont, landing_url_cont = get_dbot_std_context(
+            url_data.get('landingUrl').get('url'), 'url',
```

Comment:
no need to get `landingUrl` and `threatData` every time, do it once and then get the fields you want out of it